packages_rs/nextclade/src/align/score_matrix.rs

```diff
@@ -1,7 +1,7 @@
 use crate::align::band_2d::{Band2d, Stripe};
 use crate::align::params::{AlignPairwiseParams, GapAlignmentSide};
 use crate::alphabet::letter::Letter;
-use log::trace;
+use log::{debug, trace};
```

Comment:
Unnecessary addition
```suggestion
use log::trace;
```